Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:14-alpine
+
+WORKDIR /app
+
+RUN apk update \
+  && apk add git \
+  && git clone --recursive https://github.com/kaitai-io/kaitai_struct_webide /app \
```

Comment:
1. I don't understand why are you installing `git` into the container and then cloning the repo from inside, when this `Dockerfile` is part of the cloned repository (meaning that the `kaitai_struct_webide` repo has already been cloned)?

2. Another problem is that you'll always get the main branch (`master`) by doing this, which will make it impossible to use this `Dockerfile` for Web IDE development (like building the image from locally changed files).

   If fetching the remote `master` branch was your intent, may I ask what's the point of running the Web IDE server with Docker instead of just going to https://ide.kaitai.io/devel/ (or alternatively fetching `git clone https://github.com/kaitai-io/ide-kaitai-io.github.io` and running a local server in the `/devel/` folder).